content/modules/bpm/pages/introduction.adoc

```diff
@@ -0,0 +1,93 @@
+= Introduction
+
+
+The *BPM add-on* integrates the open-source xref:bpm:introduction.adoc#flowable-bpm[Flowable BPM engine] into *Jmix Platform*, bringing to it new tools to design, manage, and automate various complex interactions between people and systems.
+
+[[architecture]]
+== Architecture Overview
+
+*BPM add-on* includes components available in *Studio* and in the *web application* at runtime.
+In Studio, the developer has a xref:bpm:modeler-studio.adoc[*Modeler*] for creating BPMN process models, a *Process Form Wizard* (in View Creation Wizard), and a BPM section in the *Jmix Tool Window*.
+
+image::introduction/bpm-components.png[,800]
+
+In the web application, you receive the process xref:bpm:process-modeling.adoc#process-modeling-web[*Modeler*] almost the same as in Studio by functionality, a set of administrative views, and views for regular users allowing they to start processes and work with their tasks.
+
+
+Besides, some extensions become available when you model the process:
+
+* xref:bpm:bpmn/jmix-entity-data-task.adoc[*Entity Data Task*] -- allows to create, modify, and load Jmix Entities directly from the BPMN model.
```

Comment:
"allows someone to".
See https://youtrack.haulmont.com/articles/PLAT-A-1082/Tips-and-Tricks

Reply:
this block is re-writed.